src/modules/multi-file-upload/multi.file.upload.module.js

```diff
@@ -143,14 +143,14 @@ export default class MultiFileUploadModule extends Component {
      * Processes uploaded files and generates an unique file id
      *
      * @method processUploadedFiles
-     * @param {EVent} event
+     * @param {Event} event
      * @returns {Array}
      * @memberof MultiFileUploadModule
      */
-    proccessUploadedFiles(event) {
+    processUploadedFiles(event) {
         const list = this.extractDroppedFilesList(event);
 
-        return [...list.files].map((file) => ({
+        return Array.from(list.files).map((file) => ({
```

Comment:
Wouldn't it be better to use `[].slice.call(list.files).map()` instead?

Reply:
Are you thinking about browser compatibility or something else?
I checked MDN and `Array.from` seems to be supported on all modern browsers (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility) so in my opinion it's OK to use it here. 
What do you think?

Reply:
I'm thinking about perfomance

Reply:
I don't think that we need to optimise this one line. 😄 
And, for me, this notation: `Array.from(list.files)` is better than this: `[].slice.call(list.files)` when it comes to code readability. :)

Reply:
I'm fine with that